ui/src/app/routing/signed-in-app-routing.tsx

```diff
@@ -51,7 +59,7 @@ const WorkspaceSearchAdminPage = fp.flow(withRouteData, withRoutingSpinner)(Admi
 
 export const SignedInRoutes = () => {
   return <Switch>
-    <AppRoute exact path='/' guards={[expiredGuard]}>
+    <AppRoute exact path='/' guards={environment.enableDataAccessRequirements ? [expiredGuard, registrationGuard] : [expiredGuard]}>
```

Comment:
FF = false (current behavior) -> unregistered users are redirected to the homepage which **includes** the Registration Dashboard.  Thus, we cannot exclude unregistered users from the homepage.

FF = true -> unregistered users are redirected to Data Access Requirements which is **separate** from the homepage.  The homepage is now for registered users only.

Reply:
Awesome, it is much cleaner to have these routes separated.